### ISSUE TITLE
Symbol not found _exc_server in libSystem.B.dylib

### ISSUE THREAD

**Issue by Shinmera** [darlinghq/darling#929]
**Expected Result**
SBCL starts normally.

**Actual Result**
Trying to run SBCL gives:
```
dyld: Symbol not found: _exc_server
  Referenced from: /usr/local/Cellar/sbcl/2.0.11/libexec/bin/sbcl
  Expected in: /usr/lib/libSystem.B.dylib
 in /usr/local/Cellar/sbcl/2.0.11/libexec/bin/sbcl
abort_with_payload: reason: Symbol not found: _exc_server
  Referenced from: /usr/local/Cellar/sbcl/2.0.11/libexec/bin/sbcl
  Expected in: /usr/lib/libSystem.B.dylib
 in /usr/local/Cellar/sbcl/2.0.11/libexec/bin/sbcl; code: 4
Abort trap: 6 (core dumped)
```

**Steps To Reproduce**
1. Install Homebrew
2. `brew install sbcl`
3. `sbcl`

**System Information**

| Software | Version |
| --- | --- |
| Linux Kernel | 5.10.11-arch1-1 |
| Darling | f08dcae3c0b0b5da5f245099931086e02479ad9a |

**Comment by LubosD** [comment]
@facekapow This looks like a regression specific to `update-sources`. Normally, [this file](https://github.com/darlinghq/darling/blob/f08dcae3c0b0b5da5f245099931086e02479ad9a/src/kernel/libsyscall/CMakeLists.txt) should add `mach/excServer.c` (MiG generated file) to build.

**Comment by Shinmera** [comment]
Confirmed fixed on cedd1c63f48652720ee02dab6365f9e7ed1b3c39
